projects/angular/src/data/tree-view/_tree-view.clarity.scss

```
@@ -177,7 +177,9 @@
   .clr-treenode-spinner-container {
     //Dimensions
     @include mixins.equilateral(tree-view-variables.$clr-tree-node-touch-target);
-    padding: (tree-view-variables.$clr-tree-node-touch-target - tree-view-variables.$clr-tree-spinner-size) * 0.5;
+    padding: calc(
+      calc(#{tree-view-variables.$clr-tree-node-touch-target} - #{tree-view-variables.$clr-tree-spinner-size}) * 0.5
+    );
```

Try to use single calc with parentheses. It should be identical.
```suggestion
    padding: calc((#{tree-view-variables.$clr-tree-node-touch-target} - #{tree-view-variables.$clr-tree-spinner-size}) * 0.5
    );
```

---

corrected in commit 3332a98